packages/ibm-products/src/components/Checklist/Checklist.test.js

```diff
@@ -0,0 +1,187 @@
+        kind: 'checked',
+        label: 'See recommended responses',
+        onClick: (task) => {
+          console.log('clicked task', task);
+        },
+      },
+    ],
+  },
+  {
+    title: 'Explore rapid search',
+    tasks: [
+      {
+        kind: 'unchecked',
+        label: 'Search from a case',
+        onClick: (task) => {
+          console.log('clicked task', task);
+        },
+      },
+      {
+        kind: 'unchecked',
+        label: 'Run a KQL query',
+        onClick: (task) => {
+          console.log('clicked task', task);
+        },
+      },
+    ],
+  },
+  {
+    title: 'Explore threat analytics',
+    tasks: [
+      {
+        kind: 'unchecked',
+        label: 'View threat analytics dashboard',
+        onClick: (task) => {
+          console.log('clicked task', task);
+        },
+      },
+    ],
+  },
+  {
+    title: 'Manage data ingestion',
+    tasks: [
+      {
+        kind: 'unchecked',
+        label: 'Monitor data ingestion',
+        onClick: (task) => {
+          console.log('clicked task', task);
+        },
+      },
+      {
+        kind: 'unchecked',
+        label: 'Register a data ingestion point',
+        onClick: (task) => {
+          console.log('clicked task', task);
+        },
+      },
+      {
+        kind: 'unchecked',
+        label: 'See data source integrations',
+        onClick: (task) => {
+          console.log('clicked task', task);
+        },
+      },
+    ],
+  },
+];
+// calculate some values based on taskLists
+const allTasks = taskLists.map((obj) => obj.tasks).flat();
+const numTasks = allTasks.length;
+const numTasksCompleted = allTasks.filter(
+  (task) => task.kind === 'checked'
+).length;
+// std properties to use
+const chartValue = numTasksCompleted / numTasks;
+const chartLabel = `${parseInt(chartValue * 100)}% complete`;
+const title = 'Get started checklist';
+const viewAllLabel = `View all (${numTasks})`;
+
+// render a Checklist with default values and any other required props
+const renderComponent = ({ ...rest } = {}) =>
+  render(<Checklist {...{ taskLists, ...rest }} />);
+
+describe(componentName, () => {
+  it('renders a component Checklist', () => {
+    renderComponent();
+    expect(screen.getByRole('main')).toHaveClass(blockClass);
+  });
+
+  it('renders a title', () => {
+    renderComponent({ title });
+    screen.getByText(title);
+  });
+
+  it('renders the chart and chart label if `chartLabel` and `chartValue` are both defined', () => {
+    renderComponent({ chartLabel, chartValue });
+    expect(screen.getByRole('img')).toHaveClass(`${blockClass}__chart`);
+    screen.getByText(chartLabel);
+  });
+
+  it('renders a toggle', () => {
+    renderComponent({ showToggle: true, title });
+    expect(screen.getByRole('switch')).toBeDefined();
+  });
+
+  it('renders a "view all" button', () => {
+    renderComponent({ viewAllLabel });
+    screen.getByText(viewAllLabel);
+  });
+
+  it.skip('has no accessibility violations', async () => {
```

Comment:
Does this fail without the skip or is this leftover from previous n2p components?

Reply:
Apparently I've been testing individual components incorrectly all this time.

I've been using the same command as used in other projects in IBM
```
yarn jest path/to/Checklist.test.js
```

When instead I should have been using IBM Products' specific command
```
yarn test:c4p /Checklist/ 
```

The default accessibility test is passing now using the updated command.